lib/api/post-meta/functions-admin.php

```diff
@@ -103,6 +103,10 @@ function beans_register_post_meta( array $fields, $conditions, $section, $args =
  * @return bool
  */
 function _beans_is_post_meta_conditions( $conditions ) {
+	// If user has designated boolean true, it's always true. Nothing more to do here.
```

Comment:
Needs a new line above the comment.  Our coding standard is to have a new empty line above all code blocks.

Reply:
Good to know. Done.